explore/src/main/scala/explore/HelpBody.scala

```diff
@@ -36,7 +36,8 @@ final case class HelpBody(base: HelpContext, helpId: Help.Id)(implicit val ctx:
     path.segments.init.foldLeft(base.rawUrl / base.user.value / base.project.value / "main")(
       (uri, segment) => uri / segment.encoded
     )
-  private val url            = (rootUrl / "main").withPath(path)
+  private val mainUrl        = rootUrl / "main"
+  private val url            = (rootUrl / "main").withPath(mainUrl.path.concat(path))
```

Comment:
It took me a bit to understand what's going on here (and in line 79). Maybe we can extract a tiny method , say `concatToUriPath(uri: Uri, path: Uri.Path): Uri = uri.withPath(uri.path.concat(path))` ?